README.md

```diff
@@ -7,16 +7,19 @@
     [![codecov](https://codecov.io/gh/ballerina-platform/ballerina-lang/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/ballerina-lang)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Ballerina is a compiled, type safe, concurrent programming language designed to make it simple to write microservices that integrate APIs.
+In the cloud, applications interact with their environment over the network. Ballerina is a general-purpose programming language with features optimized for these network distributed applications. It is easy to write and modify and is suitable for application programmers.
 
-#### Integration Syntax
-A compiled, transactional, statically and strongly typed programming language with textual and graphical syntaxes. Ballerina incorporates fundamental concepts of distributed system integration and offers a type safe, concurrent environment to implement microservices.
+#### Services
+It has first-class language constructs for providing and consuming services,
 
-#### Networked Type System
-A type system that embraces network payload variability with primitive, object, union, and tuple types.
+#### Sequence diagrams
+The structure of functions and programs allows them to be viewed as sequence diagrams showing the interactions between concurrently executing parts.
```

Comment:
```suggestion
The structure of the functions and programs allows them to be viewed as sequence diagrams showing the interactions between the components that execute concurrently.
```

Reply:
- First edit: This isn't referring to any functions or programs, it's referring to all. Suggest not to add 'the'.
- Second edit: Suggest to keep as is, because this is the messaging reviewed by the Ballerina team.

Reply:
- First edit: You need to use "the" before a noun.
- It shouldn't be reviewed linguistically by the Ballerina team as if done it should be done by myself. Saying "concurrently-executing parts" is just shorthand (we need the hyphen there). It is better to say it in the proper way as "the components that execute concurrently."